### ISSUE TITLE
version 3.2, game crashes after loading

### ISSUE THREAD

**Issue by GameBuddy4321** [vanessakindell/wadsmoosh-freedoom#32]
GZDoom not loading the wadsmoosh+.ipk3 file. After loading the game the screen remains dark and doesn't reach main menu. Running GZDoom version: gzdoom-x64-g4.14pre-120-gfb660fa3c, Dec 05 2024 release. I installed python-3.13.1-amd64 and ran the wadsmoosh.py script to generate the combined game. 
[wadsmoosh.log](https://github.com/user-attachments/files/18046206/wadsmoosh.log)
![2024-12-06_201400](https://github.com/user-attachments/assets/15757b13-4114-458e-af06-ac00d18eeb9f)

**Comment by staticnation** [comment]
I too have this crash, I am using the mp3 versions of the sigil files (1 and 2)
[wadsmoosh.log](https://github.com/user-attachments/files/18087180/wadsmoosh.log)
[log.txt](https://github.com/user-attachments/files/18087189/log.txt)

if you notice log.txt throws an error about the zscript/wf_music.zs at line 101 (the music switching script from wadfusion)

when I explored "ipk3/music" I notice there is no "a" type for the maps
eg. there is a d_e5m1 but no d_e5m1a, etc.

line 103-104 example
	if ( music == "d_e5m1" )
		music = "d_e5m1a"; 

 same with sigil 2
 d_e6m1 but no d_e6m1a, etc.

line 180-181 example
	if ( music == "d_e6m1" )
		music = "d_e6m1a";

there is scripting for IDKFA which im not sure wadsmoosh needs since we have 3DO soundtrack support (but no 3DO soundtrack in the wf_music.zs)
 
 the plutonia music has if statements that look for h_e1m2, etc.
 folder contains p_runnin, etc. 
 
line 741-742 example
	if ( music == "h_e1m2" )
		music = "p_runnin";
		
tl;dr
so as it stands Im pretty sure that a few things need to be done to fix this:
1. the music script should be updated to reflect the 3DO soundtrack instead of the IDFKA soundtrack
2. there needs to be logic when dumping that creates the "a" variant of the music files, meaning detecting when the mp3 variant of the tracks are available in the wadsmooth_data.py and "data\music_sigil" etc.
3. the plutonia stuff needs to be redone to dump the "h_" variants as well

alternatively the "wf_music.zs" script should be stripped out of "wadsmoosh_data.py" at line 73 and "res/zscript.txt" at line 5
and the "WadFusionMusicHandler" needs to be stripped from "res\mapinfo.txt" at line 43

**Comment by Owlet7** [comment]
I would also note that WadFusionHandler does everything that WadSmooshHandler does and more. So only one of them should be used.

The XboxSecretExits class also doesn't work as intended, because the python script still edits the map files directly, changing their checksums.

**Comment by Owlet7** [comment]
The Python script copies over the custom status bar script if LoR is included, but it doesn't get set in MAPINFO because the python script can't find the right line to replace. The status bar script also gets (incorrectly) copied by copy_resources().

None of the cvars that these features use are defined.
